value_test.go

```diff
@@ -8,11 +8,63 @@ func TestExceptionString_afterClose(t *testing.T) {
 	mrb := NewMrb()
 	_, err := mrb.LoadString(`clearly a syntax error`)
 	mrb.Close()
-
 	// This panics before the bug fix that this test tests
 	err.Error()
 }
 
+func TestExceptionBacktrace(t *testing.T) {
+	mrb := NewMrb()
+	defer mrb.Close()
+
+	parser := NewParser(mrb)
+	defer parser.Close()
+	context := NewCompileContext(mrb)
+	context.SetFilename("hello.rb")
+	defer context.Close()
+
+	parser.Parse(`
+				def do_error
+					raise "Exception"
+				end
+
+				def hop1
+					do_error
+				end
+
+				def hop2
+					hop1
+				end
+
+				hop2
+			`, context)
+
+	proc := parser.GenerateCode()
+
+	_, err := mrb.Run(proc, nil)
+
```

Comment:
We're stylistic thing I can fix, nitpick: I don't like a newline between setting an error and checking it